pkgs/applications/misc/redshift-plasma-applet/default.nix

```diff
@@ -13,6 +13,13 @@ stdenv.mkDerivation {
     sha256 = "122nnbafa596rxdxlfshxk45lzch8c9342bzj7kzrsjkjg0xr9pq";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://invent.kde.org/plasma/plasma-redshift-control/-/commit/898c3a4cfc6c317915f1e664078d8606497c4049.patch";
```

Comment:
Please add a comment why we need this.

Reply:
Added the following. Included what the patch does, and when it can be removed. 

```bash

    # This patch fetches from out-of-source repo because the GitHub copy is frozen,
    #     the active fork is now on invent.kde.org. Remove this patch when a new version is released and src is updated
    # Redshift version >= 1.12 requires the -P option to clear the existing effects before applying shading.
    #     Without it scrolling makes the screen gets darker and darker until it is impossible to see anything. 
```